### DISCUSSION TITLE
TextField method to clear a inputted data.

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#899] [Q&A] [answered]
I am new to flet and I have enjoyed working with it very much, and kudos to the creators it is a very nice framework, just super easy to work with whiles yielding great results. Currently, I'm trying to create a form which will be used to collect data and when a button is clicked 
1-I want the data to be given to a database to persist. 
2-After that I want to clear the form fields so that new data can be inserted 
but I can't find a method in the form field object which does that.
So is there a method which can do that and I can't find it, or is there a workaround for the lack of it? 
Any help would be appreciated.  
Thanks in advance.

**Comment by NAKAK10** [comment, marked as answer]
🤞
```python
test_field = ft.TextField(value="aaa")

test_field.value = ""
test_field.update()
```

**Reply by Haadi-A** [reply]
Thanks, I set the values to "" but didn't know to update it.
